Resources/views/Role/list_roles.html.twig

```diff
@@ -30,10 +30,10 @@
                     <tbody>
                     {% for role in roles %}
                         {# @var role \eZ\Publish\API\Repository\Values\User\Role #}
-                        <tr>
-                            <td><a href="{{ path("admin_roleView", {"roleId": role.id}) }}">{{ role.identifier }}</a></td>
-                            <td>{{ role.id }}</td>
-                            <td>
+                        <tr class="ez-role">
+                            <td class="ez-role-name"><a href="{{ path("admin_roleView", {"roleId": role.id}) }}">{{ role.identifier }}</a></td>
+                            <td class="ez-role-id">{{ role.id }}</td>
+                            <td class="ez-role-edit-button">
```

Comment:
Looks ok with me, but maybe @dpobel has a POV on how we should sugar the output to be testable.